### DISCUSSION TITLE
It takes some time for the other node to join the RPL network. If you want an immediate response, try pinging the link-local address.

### DISCUSSION THREAD

**Discussion by fersoakd** [contiki-ng/contiki-ng#1817] [Q&A]
It takes some time for the other node to join the RPL network. If you want an immediate response, try pinging the link-local address.

I'm closing this now that things are working as they should.

_Originally posted by @atiselsts in https://github.com/contiki-ng/contiki-ng/issues/1347#issuecomment-669143945_

how can I do the Ping betwen the stations??

**Comment by atiselsts** [comment]
You can always use the ICMP API. The `uip_icmp6_send` function is what you want to send a ping message.

If you have serial access to the nodes, there's also an option to compile the code with the `shell` module. Log in to the node, and there will be `ping` command in the shell.
